### DISCUSSION TITLE
way to specify a non-default branch as production?

### DISCUSSION THREAD

**Discussion by chrisdrackett** [vercel/vercel#4182] [General] [answered]
our default branch on GitHub is `staging` so all our PR's go into that branch. This causes an issue when using the GitHub integration as we now don't have a way to set production env vars on our actual production branch (`master`). It would be nice to be able to tag a given branch (`master` in this case) our production branch.

**Comment by coetry** [comment, marked as answer]
Hi @chrisdrackett ! We have an internal card tracking this issue and will be releasing this feature soon.

**Comment by shc023** [comment]
Hello @coetry , is this available now?

**Comment by mcsdevv** [comment]
Hey @chrisdrackett, this is now available!

Further information: https://vercel.com/blog/custom-production-branch
